config_src/drivers/nuopc_cap/mom_surface_forcing_nuopc.F90

```diff
@@ -472,6 +474,16 @@ subroutine convert_IOB_to_fluxes(IOB, fluxes, index_bounds, Time, valid_time, G,
       fluxes%frunoff(i,j) = kg_m2_s_conversion * IOB%frunoff(i-i0,j-j0) * G%mask2dT(i,j)
     endif
 
+    ! add liquid glc runoff flux via rof
+    if (associated(IOB%lrunoff_glc)) then
```

Comment:
We should keep these new fields separate in fluxes and not under `fluxes%lrunoff` and `fluxes%frunoff`. 

Suggestion:
fluxes%lrunoff_glc(i,j) = fluxes%lrunoff_glc(i,j) + kg_m2_s_conversion * IOB%lrunoff_glc(i-i0,j-j0) * G%mask2dT(i,j)

fluxes%frunoff_glc(i,j) = fluxes%frunoff_glc(i,j) + kg_m2_s_conversion * IOB%frunoff_glc(i-i0,j-j0) * G%mask2dT(i,j)